src/python/macros.rs

```diff
@@ -213,7 +213,7 @@ macro_rules! impl_dual_num {
             }
 
             #[inline]
-            #[pyo3(text_signature = "($self, b: Self, c: Self)")]
+            #[pyo3(text_signature = "($self, a, b)")]
```

Comment:
wouldn't this be auto generated?

Reply:
Wasn't sure about the `$self` but yes - it is auto generated.